src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -335,9 +335,7 @@ class GroupByClauseSegment(BaseSegment):
                 Ref("ExpressionSegment"),
                 Ref("CubeRollupClauseSegment"),
                 Ref("GroupingSetsClauseSegment"),
-                Sequence(
-                    Ref("StartBracketSegment"), Ref("EndBracketSegment")
-                ),  # () possible
+                Bracketed(),  # () possible
```

Comment:
The `() possible` comment is a bit cryptic. Does this mean "Empty parentheses are possible"?

Reply:
It does. I'm going to change it. 👍